components/sections/Footer/index.tsx

```diff
@@ -0,0 +1,107 @@
+import Image from 'next/image';
+import type { FC } from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import NavItem from '@/components/sections/NavItem';
+
+import styles from './index.module.css';
+
+const Footer: FC = () => {
+  const externalLinks = [
```

Comment:
Can you actually make these part of `site.json` (https://github.com/nodejs/nodejs.org/blob/main/site.json)?

A new entry named "footerLinks" on the JSON object, and the entry should be an array of objects and the two entries are `{ link: string, text: string }`

Text by default should be a full translation id (The full `components.footer.links.XXX` not just the last part as you've done here